### PR TITLE
Feature/kas 4085 full screen decisions

### PR DESCRIPTION
**Pull request by kobemertens**

### CODE REVIEW

**Comment by kobemertens** [comment]
This should be only a few lines of changes, not sure why the diff is so large.

**Comment by ValenberghsSven** [comment]
> This should be only a few lines of changes, not sure why the diff is so large.

because it was not up to date with its base branch. I did that just now.

File: app/components/agenda/agendaitem/agendaitem-decision.hbs [hunk 1]
```diff
@@ -96,20 +96,44 @@
...
+            >{{t "update-thing" thing=(t "content")}}</AuButton>
+            <VersionsDropdown
+              @record={{this.betreftPiecePart}}
+              @previousVersionKey="previousPiecePart"
+              @onVersionClicked={{this.onRevertBetreftVersion}}
+            />
+          </:header>
+        </SayEditor>
         <h4 class="auk-h4 au-u-padding-top">{{t "agendaitem-decision"}}</h4>
-        <Agenda::Agendaitem::Decision::Editor
```
Comment (ValenberghsSven) [inline app/components/agenda/agendaitem/agendaitem-decision.hbs:106]: This component is no longer used and can be removed?
`<Agenda::Agendaitem::Decision::Editor>`

Comment (ValenberghsSven) [reply app/components/agenda/agendaitem/agendaitem-decision.hbs:106]: `Agenda::Agendaitem::Decision::PiecePartVersion` is also no longer used if the above is removed.

**Review by ValenberghsSven — COMMENTED** [review]
the versions dropdown doesn't seem to be working in fullscreen mode.
I see there is `visibility: hidden` on the `au-c-dropdown__floater` class

File: app/components/agenda/agendaitem/agendaitem-decision.hbs (hunk 1 above)
Comment (kobemertens) [reply app/components/agenda/agendaitem/agendaitem-decision.hbs:106]: Yes you're absolutely right, I forgot to remove them

Comment (kobemertens) [reply app/components/agenda/agendaitem/agendaitem-decision.hbs:106]: fixed in [ec2687f](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1784/commits/ec2687f5232923469afa09cef7f1d59c2dbe9746)

**Comment by kobemertens** [comment]
This versions dropdown is working fine on my end.
image
Can you explain what is going wrong on your end?

File: app/components/agenda/agendaitem/agendaitem-decision.js [hunk 2]
```diff
@@ -246,6 +246,12 @@ export default class AgendaitemDecisionComponent extends Component {
     this.editorInstanceBetreft = editorInterface;
   }
 
+  @action
+  onRevertBetreftVersion(betreftPiecePart) {
+    console.log(betreftPiecePart.value);
```
Comment (tdn) [inline app/components/agenda/agendaitem/agendaitem-decision.js]: Remove this log

**Review by tdn — APPROVED** [review]

**Pull request merged**